app/ide-desktop/lib/content/src/index.ts

```diff
@@ -797,6 +797,8 @@ function ok(value: any) {
 }
 
 class Config {
+    // TODO [vitvakatu]: it is a temporary feature-switch that would be removed.
```

Comment:
PRs should never contain such things. Please remove all "temporary things". Develop should be in a working stage, without elements that are "temporary"